src/reconcile/rec_col.c

```diff
@@ -1154,10 +1155,17 @@ __rec_col_var_helper(WT_SESSION_IMPL *session, WT_RECONCILE *r, WT_SALVAGE_COOKI
         val->buf.size = 0;
         val->len = val->cell_len;
     } else if (ovfl_usedp != NULL) {
+        if (__wt_process.page_stats_2022 && WT_PAGE_STAT_VALID(ovfl_ps))
```

Comment:
We should also pack in the new format even the ovfl_ps is invalid.

Reply:
Fixed here: [7fdd194](https://github.com/wiredtiger/wiredtiger/pull/8792/commits/7fdd194a5742efa7716451b29b381812ed336f2c#diff-02af988867eb44e79b42485d9a28eee9fcbbee77cf105357c6d30479ac2d028c)